tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
```

Comment:
... in which case, a comment to that effect would have been useful, since the Docker behaviour appears inconsistent.